cypress/support/commands.js

```diff
@@ -26,37 +26,36 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 //
 
-Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy^="${selector}"]`);
-});
-
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
+Cypress.Commands.add('register', (email = 'riot@qa.team',
+  username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/api/users', {
+      user: {
+        email,
+        username,
+        password
+      }
+    });
   });
-});
-
-Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  }).then(response => {
-    const user = {
-      bio: response.body.user.bio,
-      effectiveImage: "https://static.productionready.io/images/smiley-cyrus.jpg",
-      email: response.body.user.email,
-      image: response.body.user.image,
-      token: response.body.user.token,
-      username: response.body.user.username,
-    };
-    window.localStorage.setItem('user', JSON.stringify(user));
-    cy.setCookie('auth', response.body.user.token);
+  
+  Cypress.Commands.add('login', (email = 'riot@qa.team',
+  username = 'riot', password = '12345Qwert!') => {
```

Comment:
It seems like the 'login' command is defined twice in this file. The first definition is on line 14 and the second one starts here. This could lead to unexpected behavior as the second definition will overwrite the first one. Consider renaming one of them or removing the duplicate.